site/docs/use-cases/speech-processing/_sections/_usage_options/index.mdx

```diff
@@ -0,0 +1,168 @@
+import BasicGenerationConfiguration from '@site/docs/use-cases/_shared/_basic_generation_configuration.mdx';
+import GenerationConfigurationWorkflow from '@site/docs/use-cases/_shared/_generation_configuration_workflow.mdx';
+
+## Additional Usage Options
+
+:::tip
+Check out [Python](https://github.com/openvinotoolkit/openvino.genai/tree/master/samples/python/whisper_speech_recognition) and [C++](https://github.com/openvinotoolkit/openvino.genai/tree/master/samples/cpp/whisper_speech_recognition) Whisper speech recognition samples.
+:::
+
+### Use Different Generation Parameters
+
+<GenerationConfigurationWorkflow />
```

Comment:
WhisperPipeline also supports multinomial and beam search parameters. Maybe we can add some note to highlight this.
Maybe as a link to text generation where this parameters are described?

Reply:
Added multinomial and beam search config with code examples and parameters description